app/src/main/java/com/x8bit/bitwarden/ui/platform/components/model/ContentBlockData.kt

```diff
@@ -0,0 +1,30 @@
+package com.x8bit.bitwarden.ui.platform.components.model
+
+import androidx.annotation.DrawableRes
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.text.AnnotatedString
+
+/**
+ * Wrapper class for data to display in a
+ * [com.x8bit.bitwarden.ui.platform.components.content.BitwardenContentBlock]
+ */
+@Immutable
+data class ContentBlockData(
+    val headerText: AnnotatedString,
+    val subtitleText: String? = null,
+    @DrawableRes val iconResource: Int? = null,
```

Comment:
Ditto here about making sure the word `vector` gets added. Like `vectorIconResource`